src/mqtt.py

```diff
@@ -57,6 +57,7 @@ def connect(self):
 
     def disconnect(self):
         self._client.disconnect()
+        self._queue.put(None)
```

Comment:
To tell the full truth I got the idea after reading : https://github.com/monster1025/aqara-mqtt/blob/master/src/xiaomihub.py#L200
I only added the test for None in `main.py` to make sure this can never go wrong